src/global.scss

```diff
@@ -167,3 +167,12 @@ $tour-next-button-background: var(
     border: none;
   }
 }
+
+ngx-extended-pdf-viewer {
```

Comment:
nit(blocking)
I see some styles applied at the component level and some at the global level - do we need these both? Can we put everything in the component styles or do we need to override some over style applied somewhere?

Reply:
These styles are not applied when put in the component styles file unfortunately. I'm not entirely sure why, but the `!important` flag is also required for them to take effect, about which there is a small note [here](https://pdfviewer.net/extended-pdf-viewer/smartphone), under "_Caveat_"

Reply:
I think a more elegant solution might just be to remove the shadow dom encapsulation so deep selectors not required. I'll test locally and add a commit